### ISSUE TITLE
Problem with tuning `step_tokenize_bpe()`

### ISSUE THREAD

**Issue by juliasilge** [tidymodels/textrecipes#239]
The [documentation for `step_tokenize_bpe()` indicates](https://textrecipes.tidymodels.org/reference/step_tokenize_bpe.html) that `vocabulary_size` can and in fact should be tuned, but something isn't hooked up quite right:

``` r
library(tidymodels)
library(textrecipes)

data("tate_text")
folds <- vfold_cv(tate_text, v = 3)

tate_rec <- recipe(year ~ title, data = tate_text) |> 
    step_tokenize_bpe(title, vocabulary_size = tune()) |> 
    step_tf(title)

workflow(tate_rec, linear_reg()) |> tune_grid(resamples = folds)
#> Warning: No tuning parameters have been detected, performance will be evaluated
#> using the resamples with no tuning. Did you want to [tune()] parameters?
#> → A | error:   You cannot `prep()` a tuneable recipe. Argument(s) with `tune()`: 'vocabulary_size'. Do you want to use a tuning function such as `tune_grid()`?
#> There were issues with some computations   A: x1
#> There were issues with some computations   A: x3
#> 
#> Warning: All models failed. Run `show_notes(.Last.tune.result)` for more
#> information.
#> # Tuning results
#> # 3-fold cross-validation 
#> # A tibble: 3 × 4
#>   splits              id    .metrics .notes          
#>   <list>              <chr> <list>   <list>          
#> 1 <split [2856/1428]> Fold1 <NULL>   <tibble [1 × 3]>
#> 2 <split [2856/1428]> Fold2 <NULL>   <tibble [1 × 3]>
#> 3 <split [2856/1428]> Fold3 <NULL>   <tibble [1 × 3]>
#> 
#> There were issues with some computations:
#> 
#>   - Error(s) x3: You cannot `prep()` a tuneable recipe. Argument(s) with `tune()`:...
#> 
#> Run `show_notes(.Last.tune.result)` for more information.
```

<sup>Created on 2023-07-04 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

This works fine if I do not try to mark `vocabulary_size` with `tune()`.

**Comment by EmilHvitfeldt** [comment]
Thank you!!

**Comment by EmilHvitfeldt** [comment]
Closed with #240
